### ISSUE TITLE
Use system list separator (on Windows) to separate columns in CSV template

### ISSUE THREAD

**Issue by ChHaeni** [hafl-gel/agrammon-rest#2]
Current format with primary separator ; and secondary , cannot be opened easily with Excel when system list separator is ,

**Comment by ChHaeni** [comment]
Get corresponding registry entry on Windows:

```r
 system('reg query "HKEY_CURRENT_USER\\Control Panel\\International" /v sList', intern = TRUE)
```

**Comment by ChHaeni** [comment]
Issue resolved with fd017cf91ef5f878fc3503c74e2e4a73cdca0448
